Cargo.toml

```diff
@@ -36,7 +36,13 @@ rand_core = { path = "rand_core", version = "0.2", default-features = false }
 rand_isaac = { path = "rand_isaac", version = "0.1" }
 rand_xorshift = { path = "rand_xorshift", version = "0.1" }
 log = { version = "0.4", optional = true }
-packed_simd = { version = "0.1", optional = true, features = ["into_bits"] }
+
+[dependencies.packed_simd]
+# We frequently hit breakage in this package due to instability
+# version = "0.1"
+git = "https://github.com/rust-lang-nursery/packed_simd"
```

Comment:
The current version is 0.2, you might want to just track the latest released version on crates.io.

Reply:
But this requires nightly? Okay...